app.js

```diff
@@ -221,3 +224,23 @@ submit.addEventListener('submit', (e) => {
   return true;
 });
 
+function savaData() {
+  const userData = {
+    username: fullname.value,
+    email: email.value,
+    message: message.value,
+  };
+  localStorage.setItem('data', JSON.stringify(userData));
+}
+
+fullname.addEventListener('focusout', savaData);
+email.addEventListener('focusout', savaData);
+message.addEventListener('focusout', savaData);
+
+const data = JSON.parse(localStorage.getItem('data'));
+
+if (data) {
+  fullname.value = data.username;
+  email.value = data.email;
+  message.value = data.message;
+}
```

Comment:
Please consider wrapping it in the window.load function so it checks if the data is available when the window loads for the first time to meet the project requirements ✅ 

```
window.onload = () => {

  if (data) {
    fullname.value = data.username;
    email.value = data.email;
    message.value = data.message;
  }

};
```

## REMEMBER:- 
This is just an example code consider reviewing and verifying your code if it works before submitting the request 💯